libs/dal/src/repositories/subscriber-preference/subscriber-preference.entity.ts

```diff
@@ -0,0 +1,8 @@
+export class SubscriberPreferenceEntity {
+  _subscriberId: string;
+  _templateId: string;
+  enabled: boolean;
+  channels: { email: boolean; sms: boolean; in_app: boolean; direct: boolean; push: boolean };
```

Comment:
What do you think of creating a different interface for this? And creating this interface also in package shared ?

Reply:
Love the idea I was not sure if keep it that way or do Record<ChannelTypeEnum, boolean>